src/components/sbb-expansion-panel-header/sbb-expansion-panel-header.scss

```diff
@@ -25,7 +25,7 @@
   @include sbb.focus-outline;
 
   outline-offset: var(--sbb-spacing-fixed-1x);
-  border-radius: var(--sbb-expansion-panel-border-radius, var(--sbb-border-radius-4x));
+  border-radius: var(--sbb-expansion-panel-border-radius);
```

Comment:
sorry, was proposing it like it was in last review :D. but components are so tightly coupled, that we can agree this.